packages/desktop/components/popups/AllotManaPopup.svelte

```diff
@@ -104,7 +104,7 @@
             const accountId = AddressConverter.parseBech32Address(accountAddress)
             // Send 0 amount transaction to accountAddress with amount in the allotMana
             const prepareOutput = await $selectedWallet.prepareOutput(
-                { recipientAddress: accountAddress, amount: '0' },
+                { recipientAddress: $selectedWallet.depositAddress, amount: '0' },
```

Comment:
This way you are still generating a basic output with a SDRUC, is there no other way to make this so the account gets transitioned and thats all? See the transaction `/alphanet/transaction/0x7b29a68e275e771f4f46e9b7bcde6fa943625c61b919b6155375e13841b79fa427810100`. Sounds like something we could ask the SDK team if they have an idea of how to avoid these random basic outputs with SDRUC